vllm/attention/backends/placeholder_attn.py

```diff
@@ -316,9 +316,14 @@ def build(self, seq_lens: List[int], query_lens: List[int],
                                  -1 if cuda graph is not used.
             batch_size: The maybe padded batch size.
         """
-        for inter_data in self.input_builder.inter_data_list:
-            self._add_seq_group(inter_data,
-                                self.input_builder.chunked_prefill_enabled)
+
+        # Some input builders such as ModelInputForCPUBuilder do not have the
+        # "inter_data_list" attribute.
+        # Let's check inter_data_list exists before we reference it.
+        if hasattr(self.input_builder, "inter_data_lis"):
```

Comment:
```suggestion
        if hasattr(self.input_builder, "inter_data_list"):
```

Just noticed a typo here

Reply:
great catch!

Reply:
Fixed with last series of commits